package.json

```diff
@@ -20,6 +20,7 @@
     "prismjs": "^1.28.0",
     "react": "^17.0.2",
     "react-cookie": "^4.1.1",
+    "react-dark-mode-toggle-2": "^2.0.7",
```

Comment:
It looks like this library is actually quite large (when considering its dependencies). Perhaps it would be better to just go with a simple checkbox.